infrastructure/modules/scaling_service/main.tf

```diff
@@ -10,7 +10,7 @@ locals {
 }
 
 module "service" {
-  source = "git::github.com/wellcomecollection/terraform-aws-ecs-service.git//modules/service?ref=v3.13.1"
+  source = "git::github.com/wellcomecollection/terraform-aws-ecs-service.git//modules/service?ref=rk_add-entrypoint"
```

Comment:
I suspect this needs to change back.

Reply:
Yeppers, but that branch needs merging and releasing first 👍

Reply:
There are a few more `rk_add-entrypoint`s elsewhere. I recall this was some kind of automated IDE change or similar last time.

Reply:
PR merged and released: https://github.com/wellcomecollection/terraform-aws-ecs-service/releases/tag/v4.1.0, updated module references to match.